service.subtitles.legendastv/service.py

```diff
@@ -0,0 +1,304 @@
+# -*- coding: UTF-8 -*-
```

Comment:
keep in mind for a future update that this file (as the addon entrypoint) should be kept to a minimum. Ideally you only import your main logic (stored in a module) and execute it. Python entrypoints do not get compiled to bytecode by kodi's python interpreter and this fact may reduce the performance of the addon.

You should have something like this in the entrypoint:

```python
import yourmodule

yourmodule.run()
```

All the logic should then be kept in `yourmodule.py`